src/main/java/com/cgessinger/creaturesandbeasts/entities/LittleGrebeEntity.java

```diff
@@ -59,8 +59,6 @@
 import javax.annotation.Nullable;
 import java.util.UUID;
 
-import net.minecraft.world.entity.AgeableMob.AgeableMobGroupData;
-
 public class LittleGrebeEntity extends Animal implements IAnimatable {
     public static final Ingredient TEMPTATION_ITEMS = Ingredient.of(Items.COD, Items.SALMON, Items.TROPICAL_FISH);
```

Comment:
Still doing a weird thing here with removing an import, but also you didn't actually change the temptation items here to a tag.

Reply:
Maybe cherry picking wasn't the best way to go about things for this PR lol

Reply:
Hahaha you’re fine! Sorry things have been so slow on this. I’m finally getting to updating things. Basically just waiting on these PRs to be ready!

Reply:
Okay, now hopefully that's everything fixed